proposal.tex

```diff
@@ -37,6 +48,17 @@ \subsection{BPS Batch }\label{sec:bpsbatch}
 This would also cover DMS-REQ-0119, DMS-REQ-0125, DMS-REQ-0128, DMS-REQ-0123, DMS-REQ-0127.
 
 
+\subsubsection{Non Rubin code} \label{sec:othercode}
+Users may wish to run arbitrary code on Rubin images.
+It should be possible to provide a butler recipe to get an image or set of images to the users space and allow them to run any code which takes FITS images.
+Done in a container this could be done for a large volume of images - it would have to get past  the resource allocation committee of course.
+This would require custom work by the user and would not be a general facility.
+Run in this mode little provenance would be kept.  (See \secref{sec:gen3})
+
+Some provenance could be garnered by ingesting the generated files which could at least give the execution graph of their creation.
+If users wish only to write files they would have to go in their user space or potentially a VOSpace end point - the user would need to organize and track such files themselves.
+
+
 \subsection{Arbitration or Resource Allocation Committee}\label{sec:arbitration}
 As \secref{sec:resources} points out we will need an arbitration committee of some sort.
 It seems this should also favor individuals from under represented  and under resourced groups though that is being proposed here for the first time.
```

Comment:
I like this idea and it is consistent with larger early justifications for the LSST project.

Note also that we have said publicly that proposals committing to releasing Level 3 data products that are made available to all data rights holders will be allowed to be given a preference by the RAC - as part of the overall mix of other considerations including scientific value, not an absolute requirement.  That is perhaps worth repeating here.

Reply:
line added